frontend/src/pages/PropertyDetails.jsx

```
@@ -586,228 +739,88 @@ const PropertyDetails = () => {
+                                        value={comment}
+                                        onChange={(e) => setComment(e.target.value)}
+                                        rows={3}
+                                        placeholder="Write your comment here..."
+                                        className="w-full border rounded-md p-2 focus:outline-none focus:ring-2 focus:ring-blue-500 text-sm mb-2"
+                                    ></textarea>
+                                    {submitError && <p className="text-red-500 text-sm mb-2">{submitError}</p>}
+                                    <button
+                                        onClick={handleSubmitReview}
+                                        className="bg-blue-600 text-white px-4 py-2 rounded-md hover:bg-blue-700 transition"
+                                    >
+                                        Submit Review
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)